pkg/core/mempool/prometheus.go

```diff
@@ -11,14 +11,29 @@ var (
 			Namespace: "neogo",
 		},
 	)
+	// notarypoolUnsortedTx prometheus metric.
+	notarypoolUnsortedTx = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Help:      "Notary request pool fallback txs",
+			Name:      "notarypool_unsorted_tx",
+			Namespace: "neogo",
+		},
+	)
 )
 
 func init() {
 	prometheus.MustRegister(
 		mempoolUnsortedTx,
+		notarypoolUnsortedTx,
 	)
 }
 
-func updateMempoolMetrics(unsortedTxnLen int) {
+// UpdateMempoolMetrics updates metric of the number of unsorted txs inside the mempool.
+func UpdateMempoolMetrics(unsortedTxnLen int) {
```

Comment:
Maybe we can move these metrics out of `mempool` into `core` and `network` respectively?

Reply:
Yes, looks better this way, fixed.